### ISSUE TITLE
Consumer product recall has text missing to identify the purpose of the section

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#126]
URL: https://hcdev.openplus.ca/en/alert-recall/test-alert-oct-13-new-recall
User Agent: Chrome 94.0
Language: Both 

Issue Description:  The section underneath 'What to do' is missing text to provide the user with the purpose of the section.  I think it's meant to be 'Audience'.

Steps to reproduce: Visually scan the page using the url provided above.
![image](https://user-images.githubusercontent.com/68117281/137540954-e5207158-b13b-49dd-8a46-51035a78adc7.png)

Expected results:  The page should have structured information so that the user has some context with the information being presented

**Comment by echouin** [comment]
@ndolter Quick note on this: The way the "Who is this for" is supposed to work is that it should only appear in the summary if "General Public" is Not selected. In that case it's used to help the public know that this recall isn't targeted at them.

**Comment by echouin** [comment]
From OP: Completed.

**Comment by hydegeoff** [comment]
Note: exact article no longer exists.  Similar found.  Tested in PROD.

"Audience:" has been added to section title.  ticket closed.
